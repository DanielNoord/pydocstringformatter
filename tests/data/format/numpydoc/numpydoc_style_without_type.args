--style=numpydoc
--no-final-period
--no-closing-quotes
