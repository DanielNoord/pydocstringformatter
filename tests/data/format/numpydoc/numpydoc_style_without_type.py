def sincos(theta):
    """Returns
    ----
    sin: float
        the sine of theta
    cos: float
        the cosine of theta
    Parameters
    -----
    theta:
        the angle at which to calculate the sine and cosine.
"""
    return math.sin(theta), math.cos(theta)
