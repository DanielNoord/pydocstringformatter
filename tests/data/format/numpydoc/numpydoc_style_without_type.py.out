def sincos(theta):
    """Parameters
    -------------
    theta:
        the angle at which to calculate the sine and cosine.

    Returns
    -------
    sin : float
        the sine of theta
    cos : float
        the cosine of theta
    """
    return math.sin(theta), math.cos(theta)
